### PR TITLE
Solidity 0.8.13 compiler

### PR DESCRIPTION
**Pull request by alexroan**

### CODE REVIEW

File: contracts/test/v0.8/dev/ArbitrumSequencerUptimeFeed.test.ts [hunk 1]
```diff
@@ -306,7 +306,7 @@ describe('ArbitrumSequencerUptimeFeed', () => {
       // Assert update
       expect(await arbitrumSequencerUptimeFeed.latestAnswer()).to.equal(1)
       expect(updateTx.cumulativeGasUsed.toNumber()).to.be.closeTo(
-        93137,
+        93015,
```
Comment (RyanRHall) [inline contracts/test/v0.8/dev/ArbitrumSequencerUptimeFeed.test.ts:309]: does this belong in this PR?

**Review by RyanRHall — APPROVED** [review]

File: contracts/test/v0.8/dev/ArbitrumSequencerUptimeFeed.test.ts (hunk 1 above)
Comment (alexroan) [reply contracts/test/v0.8/dev/ArbitrumSequencerUptimeFeed.test.ts:309]: Yes, Interestingly. Hardhat uses the highest supported Solidity version within a contract's defined pragma to run its tests against. 

It turns out this contract has a wide pragma, so these tests used to run on a 0.8.6 compiled contract, and now get run on 0.8.13, which has some gas optimizations.

Comment (RyanRHall) [reply contracts/test/v0.8/dev/ArbitrumSequencerUptimeFeed.test.ts:309]: :+1:

**Review by makramkd — APPROVED** [review]
:shipit:

**Pull request merged**